lib/serviceBusMessage.ts

```diff
@@ -198,23 +198,23 @@ export module SendableMessageInfo {
       throw new Error("'msg' cannot be null or undefined.");
     }
 
-    if (msg.contentType && typeof msg.contentType !== "string") {
+    if (msg.contentType != undefined && typeof msg.contentType !== "string") {
```

Comment:
Why the move from `if (a)` to `if (a != undefined)`? Same for other checks in this file that are updated in this file. Are we trying to catch a particular case here?

Reply:
Empty strings are allowed which the previous checks will not allow. Hence updating the code to specifically check for null and undefined.

Reply:
I thought so :)

I usually prefer something like `msg.hasOwnProperty('contentType')` in such cases. Thoughts?